t/chaos/killetcd/killetcd.go

```diff
@@ -140,17 +140,12 @@ var _ = ginkgo.Describe("Test Get Success When Etcd Got Killed", func() {
 	// fail to set route since etcd is all killed
 	// while get route could still succeed
 	ginkgo.It("get stats after kill etcd", func() {
-		timeStart := time.Now()
 		utils.SetRoute(e, httpexpect.Status5xx)
 		utils.GetRoute(eDataPanel, http.StatusOK)
 		utils.TestPrometheusEtcdMetric(ePrometheus, 0)
 
 		bandwidthAfter, durationAfter = utils.GetEgressBandwidthPerSecond(ePrometheus)
 		bpsAfter = bandwidthAfter / durationAfter
-
-		errorLog, err := utils.Log(apisixPod, cliSet.KubeCli, timeStart)
```

Comment:
Skip this test because it fails because it's a conf server specific test.
Note that we must keep this test even if we do not execute it, because it shares some data with the next test.

```
Error: 023/08/19 08:19:53 [error] 46#46: *9103 [lua] exporter.lua:456: handler(): prometheus: failed to reach config server while processing metrics endpoint: has no healthy etcd endpoint available
```